runner/copy.go

```diff
@@ -17,6 +17,9 @@ import (
 	"go.uber.org/zap"
 )
 
+const resourceRecordBufferSize = 1_000_000
+const waitGroupSize = 1000
```

Comment:
I'd add a comment and make this workerSize

Make it clear that this is the number of go-rountines PER materialization chunk runner to make redis requests